### PR TITLE
Alphapolis

### LINKED ISSUE #93: Add mangapolice
https://www.alphapolis.co.jp/manga/official/44000215

### PR DESCRIPTION
**Pull request by Robonau**
resolves #93

### CODE REVIEW

**Comment by Robonau** [comment]
an improvement was suggested in dms

File: src/web/mjs/connectors/Alphapolis.mjs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import Manga from '../engine/Manga.mjs';
+
+export default class Alphapolis extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'alphapolis';
+        super.label = 'ALPHAPOLIS (アルファポリス)';
+        this.tags = ['manga', 'japanese', 'hentai'];
+        this.url = 'https://www.alphapolis.co.jp/';
```
Comment (27136773) [inline src/web/mjs/connectors/Alphapolis.mjs]: Remove trailing `/`

File: src/web/mjs/connectors/Alphapolis.mjs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        `;
+        const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        const data = await Engine.Request.fetchUI(request, script);
```
Comment (27136773) [inline src/web/mjs/connectors/Alphapolis.mjs]: No need for `data`, may use `return Engine.Request.fetchUI` directly

**Review by 27136773 — APPROVED** [review]
Looks good

File: src/web/mjs/connectors/Alphapolis.mjs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        `;
+        const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        return await Engine.Request.fetchUI(request, script);
```
Comment (27136773) [inline src/web/mjs/connectors/Alphapolis.mjs:50]: No need to use await, a `Promise` is fully compatible with async, in fact async function promisifies the result:
"_If the result is a promise, return it, if not wrap the result in a promise and return it_"

**Pull request merged**